.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,41 @@
+name: New issue for Reproducible Bug
+about: "If you're sure it's reproducible and not just your machine: submit an issue so we can investigate."
+labels: bug
+issue_body: false
+inputs:
+  - type: description
+    attributes:
+      value: Please note we will close your issue without comment if you do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+  - type: checkboxes
+    attributes:
+      label: Contributor Checklist
+      description: Please verify that you've followed these steps
+      choices:
+        - label: I ran `brew update` and can still reproduce the problem?
```

Comment:
There are converting to inline code directly. Perhaps the template doesn’t yet support Markdown and we’d have to use HTML?

Reply:
@vitorgalvao I think it's fine even without but, yeh, good pointing that out.